client/my-sites/plans-grid/components/sticky-container.tsx

```diff
@@ -86,7 +95,7 @@ export function StickyContainer( props: Props ) {
 				ref={ stickyRef }
 				stickyOffset={ stickyOffset }
 				disabled={ disabled }
-				className={ isStuck ? stickyClass : '' }
+				className={ classNames( { [ stickyClass ]: isStuck }, className ) }
```

Comment:
Is `stickyClass` not obsolete now since we are passing `className`?

Reply:
Thanks for the ping @jdc91! I could've been clearer here. This comment is addressed in https://github.com/Automattic/wp-calypso/pull/85520#discussion_r1435439562

`stickyClass` is only applied to the component if it is actually "stuck" to the screen. `className` was meant to allow the consumer of the `StickyContainer` to apply a `className` that would be present on the container, regardless of its "sticky" state. After the revisions made here https://github.com/Automattic/wp-calypso/pull/85520#discussion_r1440110172, however, and the shift away from using media queries, `className` itself is no longer needed